account_invoice_validation/models/res_partner.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    validation_user_id = fields.Many2one(
+        "res.users",
+        "Approver user",
+        help="User in charge of the invoices/refunds approval.",
+        company_dependent=True,
+    )
+
+    validation_user_id_domain = fields.Binary(
+        string="approver user domain",
+        help="Dynamic domain for approver user",
+        compute="_compute_validation_user_id_domain",
+    )
+
+    use_invoice_first_approval = fields.Binary(
+        help="Use a first level of approbation: approver can be set on vendors",
+        compute="_compute_use_invoice_first_approval",
+    )
+
+    def _compute_use_invoice_first_approval(self):
+        self.use_invoice_first_approval = (
```

Comment:
IMO it should be something like
`self.update({"use_invoice_first_approval": self.env["ir.config_parameter"].sudo().get_param("account_invoice_validation.use_invoice_first_approval")})`.

In case of multi-create, that could break.